commit b3f5aec539af39cbc2006b7af5bee25752a60b68 (refs/changes/74/974/1)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2016-11-29T15:12:28-08:00 (2 years, 10 months ago)
    
    AGENT-1048 vm-agent builds missing libgcc required to work around platform

diff --git a/Makefile b/Makefile
index f52b705..848d3b7 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ JSSTYLE_FLAGS =
 
 # Should be the same version as the platform's /usr/node/bin/node.
 NODE_PREBUILT_TAG =	gz
-NODE_PREBUILT_VERSION =	v4.6.0
+NODE_PREBUILT_VERSION =	v4.6.1
 ifeq ($(shell uname -s),SunOS)
 	# Allow building on a SmartOS image other than sdc-smartos/1.6.3.
 	NODE_PREBUILT_IMAGE =	18b094b0-eb01-11e5-80c1-175dac7ddf02
@@ -103,8 +103,10 @@ release: all deps docs $(SMF_MANIFESTS)
 	rm -rf $(RELSTAGEDIR)/$(NAME)/node_modules/kthxbai \
 	    $(RELSTAGEDIR)/$(NAME)/node_modules/.bin/kthxbai
 	uuid -v4 > $(RELSTAGEDIR)/$(NAME)/image_uuid
-	mkdir -p $(RELSTAGEDIR)/$(NAME)/node/bin
+	mkdir -p $(RELSTAGEDIR)/$(NAME)/node/bin $(RELSTAGEDIR)/$(NAME)/node/lib
 	cp $(NODE_INSTALL)/bin/node $(RELSTAGEDIR)/$(NAME)/node/bin/
+	cp -RP $(NODE_INSTALL)/lib/* $(RELSTAGEDIR)/$(NAME)/node/lib/
+	rm -rf $(RELSTAGEDIR)/$(NAME)/node/lib/node_modules
 	cd $(RELSTAGEDIR) && $(TAR) -zcf $(TOP)/$(RELEASE_TARBALL) *
 	cat $(TOP)/manifest.tmpl | sed \
 	    -e "s/UUID/$$(cat $(RELSTAGEDIR)/$(NAME)/image_uuid)/" \
